Update CAIO_LEONARDO w/ Atividade07 (Aula08)

CaioSR opened the pull request.

CaioSR committed:
Update CAIO_LEONARDO w/ Atividade07 (Aula08)

CaioSR committed:
Update Dupla_CAIO_LEONARDO.md

filipesaraiva commented on duplas/Dupla_CAIO_LEONARDO.md:
```diff
@@ -3,8 +3,18 @@
 - Atividade05: [link aqui](https://docs.google.com/document/d/1OJC4tPS-wKc1dyBgXqeB0EW4DIO4p8uDfMZCxs7g4Cg/edit?usp=sharing)
+- Atividade06: #
+
+### - Atividade07 (Aula 08.md)
```
Aparentemente hã 2 atividades realizadas, faltando portanto 1.

filipesaraiva replied:
Essa parte está ok.

CaioSR replied:
Tem 3 professor. Definição do Linter (1), boas práticas para linguagens (2), e boas práticas para frameworks (3).

filipesaraiva commented on duplas/Dupla_CAIO_LEONARDO.md:
```diff
@@ -3,8 +3,18 @@
 - Atividade02: [link aqui](https://docs.google.com/document/d/1LwzV5GmNyDzqV2em_NOY0ImchHwWyPy5B-sRz0WOY3M/edit?usp=sharing)
 - Atividade03: #
 - Atividade04: [link aqui](https://docs.google.com/document/d/1NoUil0i90oIbWwlwSgzAz_8b5lgh7545YNtIgb60OE0/edit?usp=sharing)
 - Atividade05: [link aqui](https://docs.google.com/document/d/1OJC4tPS-wKc1dyBgXqeB0EW4DIO4p8uDfMZCxs7g4Cg/edit?usp=sharing)
```
Colocar o conteúdo dessas atividades diretamente no arquivo, e não mais em documentos externos no docs.

CaioSR replied:
Pronto.

CaioSR committed:
Adjusted AND added Atividade08 (Aula 9.md)

Retirei os arquivos externos (exceto pela atividade 01 que foi respondida nas spreadsheets do google conforme instruído) e adicionei a atividade 08.
